docs/quick-start.md

```diff
@@ -0,0 +1,84 @@
+
+It helps to keep recordings as short as possible – focus on the one interaction you want to understand better.
+
+When you finish recording, CodeSee will open a new tab for your recording once all the data has been transmitted. Larger recordings or slower networks can cause some buffering, so please wait for it to finish sending the recording to our servers.
+
+- Note: If the delay is long enough, your browser's pop-up blocker may decide to block the page from displaying. You may want to change your settings to always allow pop-ups from the app.codesee.io domain. If the page is blocked, you can always go to [app.codesee.io/library](https://app.codesee.io/library) to see all your latest recordings.
+
+## Understanding data flows
+
+A data flow is a detailed timeline view of the code that ran during a CodeSee recording. The data flow is laid out in code execution order, meaning that it shows what order each function was called in regardless of what file the function is located in.
+
+![CodeSee data flow](../img/data_flow.png)
+
+**Source code**
+
+- Source code that executed is displayed in the left-most column.
+- The function name is located at the top left and the file path is located to the right.
+- Any code that was skipped during execution is shown grayed out.
+
+**Runtime data**
+
+* The data flow shows expressions in the center column and the value of each of the expressions at the time the recording was made in the right most column.
+* When you hover over the expression in the center column it highlights the expression in the code in the left most column.
+* When a function calls another function, you can dive in by clicking on the expand button `>` in the middle column to view the nested function.
+
+**Loops**
+
+* If there is a loop, whether it’s an explicit loop like a `for` loop or an implicit loop like `reduce` you can visualize each iteration of the loop and the values of the data on each loop’s iteration.
+
+**Call stack**
+
+* When you are diving into a data flow the current function that you are exploring sticks to the top so you always are aware of what line of code is part of which function. If more than 3 functions are in the stack, a message appears to say how many functions are hidden.
+
+**Side Effects**
+
+* When you click on the "View" button in the center column for a side effect, such as a network request, an inspector panel appears in the data flow. Here you can inspect details about any side effects that occurred.
+
+**Filtering**
+
+* You can hover over any function name and click on the `•••` button to filter out and hide all functions of that type by selecting "Exclude All - Filter out all instances."
+
+
+## Using stories to capture your understanding
+
+Stories allow you to annotate a data flow recording and add notes that are anchored to specific points in the code execution.
+
+To create or view a story, open the story bar on the far right of the data flow screen.
```

Comment:
For the future: probably want to add details about how to click the link to jump to the code.